Cargo.toml

```diff
@@ -23,9 +23,10 @@ ini = ["rust-ini"]
 json5 = ["json5_rs"]
 
 [dependencies]
+async-trait = "0.1.50"
 lazy_static = "1.0"
-serde = "1.0.8"
 nom = "6"
+serde = "1.0.8"
```

Comment:
This diff is also not necessary, please revert it (moving the `serde` line).

Reply:
Done